input.lua

```diff
@@ -175,6 +175,87 @@ end
 
 function courseplay:executeFunction(self, func, value, page)
 	courseplay:debug("executeFunction: function: " .. func .. " value: " .. tostring(value) .. " page: " .. tostring(page), 5)
+	local validateClasses={
```

Comment:
All CpObject derived classes have an is_a() function, that's may be easier to use.

Reply:
I will have a look at that